src/general_manager/permission/mutationPermission.py

```
@@ -0,0 +1,88 @@
+from __future__ import annotations
+from django.contrib.auth.models import AbstractUser, AnonymousUser
+from typing import Any, Dict, Optional, Type
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)